pkg/detector/detector.go

```diff
@@ -732,7 +726,8 @@ func (d *ResourceDetector) ClaimPolicyForObject(object *unstructured.Unstructure
 	objLabels := object.GetLabels()
 	if objLabels == nil {
 		objLabels = make(map[string]string)
-	} else if len(objLabels) > 0 {
+	}
+	if len(objLabels) > 0 {
```

Comment:
Why?

Reply:
Reduce one layer of cyclomatic complexity without changing the logic.

Reply:
Probably not, if-else is different from `if-if`, and the cyclomatic is not reduced.

Reply:
You're right. The cyclomatic complexity of using `if` and `else if` is the same.